quickwit/quickwit-cli/src/cli.rs

```diff
@@ -37,10 +37,10 @@ pub fn build_cli<'a>() -> Command<'a> {
                 .default_value(DEFAULT_QW_CONFIG_PATH)
                 .global(true),
         )
-        .subcommand(build_run_command().display_order(1))
-        .subcommand(build_index_command().display_order(2))
-        .subcommand(build_source_command().display_order(3))
-        .subcommand(build_split_command().display_order(4))
+        .subcommand(build_index_command().display_order(1))
+        .subcommand(build_source_command().display_order(2))
+        .subcommand(build_split_command().display_order(2))
+        .subcommand(build_run_command().display_order(3))
```

Comment:
`run`, `index`, `source`, `split` makes more sense.

Reply:
Ok, it was already this display order before my update. Maybe I didn't understand the task:

- `Fix the display order of some commands (1. config, 2. index, 3. source/split, 4. other options)` 

Is there any modifications to make for this task ?

Reply:
Ah well.... I guess I disagree with @guilload.

I'm ok with that order then.

Reply:
(apologies for the noise)

Reply:
You changed `1, 2, 3, 4` to `1, 2, 2, 3`.

Reply:
Regarding the order of the commands, I did not express myself correctly. I mean the order of the options in the commands: `1. config, 2. index, 3. checkpoint/source/split, 4. other options/flags`.